template/src/index.js

```diff
@@ -1,22 +1,38 @@
 import 'core-js/stable'
 import 'regenerator-runtime/runtime'
-import { registerHtml, start } from 'tram-one'
-import ColorHeader from './components/ColorHeader'
+import { registerHtml, start, useEffect } from 'tram-one'
+import AppHeader from './app-header'
+import AppSummary from './app-summary'
+import appTaskDescription from './app-task-description'
+import AppTaskList from './app-task-list'
 import './styles.css'
-import useColor from './hooks/useColor'
+
+/**
+ * The entry point of the app, and where we mount the app on the DOM.
+ * Read more about it here: https://tram-one.io/#components
+ */
 
 const html = registerHtml({
-	ColorHeader
+	'app-header': AppHeader,
```

Comment:
indentation?

Reply:
previously the project inconsistently has tabs and spaces. It should now all be tabs (this will be better enforced with a linting tool).